tox.ini

```diff
@@ -39,6 +39,7 @@ changedir =
   test-ext-jaeger: ext/opentelemetry-ext-jaeger/tests
   test-ext-pymongo: ext/opentelemetry-ext-pymongo/tests
   test-ext-wsgi: ext/opentelemetry-ext-wsgi/tests
+  test-ext-zipkin: ext/opentelemetry-ext-zipkin/tests
```

Comment:
You'll have to rebase this now that #291 is in.

Reply:
Done